eng/Versions.props

```diff
@@ -44,19 +44,19 @@
     <MicrosoftNETBuildExtensionsPackageVersion>$(MicrosoftNETSdkPackageVersion)</MicrosoftNETBuildExtensionsPackageVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview8-28379-02</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>5.0.0-alpha1.19402.7</MicrosoftNETCoreAppPackageVersion>
     <MicrosoftNETCoreAppInternalPackageVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreAppInternalPackageVersion>
     <MicrosoftNETCoreDotNetAppHostPackageVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreDotNetAppHostPackageVersion>
     <MicrosoftNETCoreDotNetHostResolverPackageVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreDotNetHostResolverPackageVersion>
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
     <NetCoreAppTargetingPackVersion>$(MicrosoftNETCoreAppPackageVersion)</NetCoreAppTargetingPackVersion>
     <NetCoreAppHostPackVersion>$(MicrosoftNETCoreAppPackageVersion)</NetCoreAppHostPackVersion>
-    <NETStandardLibraryRefPackageVersion>2.1.0-preview8-28379-02</NETStandardLibraryRefPackageVersion>
+    <NETStandardLibraryRefPackageVersion>2.1.0-alpha1.19402.7</NETStandardLibraryRefPackageVersion>
```

Comment:
FYI, @dagood this is a downgrade. I don't think it matters for now.

Reply:
We should maybe stop producing new netstandard out of master.

Reply:
It seems like this can (should?) be fixed without doing that though, by changing the `Version.Details.xml` for this particular package to use the 3.0 builds. I filed https://github.com/dotnet/core-setup/issues/7567 to track upgrading the version of this package that core-setup produces.

Reply:
How would I do that? Mark it as "pinned"?

Reply:
I think I was mistaken... I don't see a way to split a subscription per package, or specify some sort of binding in the `Version.Details.xml`. Not sure what I was thinking, maybe jumbling in the old `dependencies.props`. 😕

Pinning would make it not *change*, but it doesn't seem ideal for the 5.0 branch to get behind the 3.0 branch. I think we should just not worry about it and https://github.com/dotnet/core-setup/issues/7567 will bring the package version forward. Depending on 3.0 might have been a bad idea anyway, not sure what the expectations are for 5.0.

/cc @mmitche if I'm forgetting something that makes it possible.

Reply:
Yes, I think we can leave it and let it become 2.2 sooner or later.

This does raise a bigger question though. Do we always expect to have exactly one netstandard targeting pack bundled and is it always going to get reved 1:1 with .NET Core releases?

Reply:
Big question... I filed https://github.com/dotnet/standard/issues/1403 to discuss, I haven't seen much info on this topic. It came up in the SDK simplification effort too IIRC, specifically if we'll need to bundle more than one.